wandb_preempt/checkpointer.py

```diff
@@ -176,12 +176,18 @@ def save_checkpoint(self, extra_info: Dict) -> None:
         save(data, tmp_savepath)
         rename(tmp_savepath, savepath)
 
-    def load_latest_checkpoint(self) -> Tuple[int, Dict]:
+    def load_latest_checkpoint(
+        self, weights_only: bool = True, **kwargs
+    ) -> Tuple[int, Dict]:
         """Load the latest checkpoint and set random number generator states.
 
         Updates the model, optimizer, lr scheduler, and gradient scaler states
         passed at initialization.
 
+        Args:
+            weights_only: Whether to load only the model weights. Default: `True`.
```

Comment:
I think this is too narrow, e.g. use 'Indicates whether unpickler should be restricted to loading only tensors, primitive types, dictionaries', see the docstring from https://pytorch.org/docs/stable/generated/torch.load.html

Reply:
True, will rephrase